addons/binding/org.openhab.binding.autelis/src/main/java/org/openhab/binding/autelis/internal/discovery/AutelisDiscoveryParticipant.java

```diff
@@ -35,7 +35,8 @@
     private final Logger logger = LoggerFactory.getLogger(AutelisDiscoveryParticipant.class);
 
     private static String MANUFACTURER = "autelis";
-    private static String MODEL = "pc100pi";
```

Comment:
No support anymore for `pc100pi`, or is it capture by a prefix so now by 'pc100p'

Reply:
Correct